lib/services/addon-service.js

```diff
@@ -0,0 +1,114 @@
+'use strict';
+
+const memoize = require('memoizee');
+const addonProvidersClasses = require('../addons');
+const events = require('../event-type');
+const { addonSchema, addonDefinitonSchema } = require('./addon-schema');
+
+const SUPPORTED_EVENTS = Object.keys(events).map(k => events[k]);
+
+const ADDONS_CACHE_TIME = 60 * 1000; // 60s
```

Comment:
Does it make sense to cache all addon configurations for 60 seconds?

Reply:
I'd say so. At least having a cache sounds reasonable. We mentioned being able to survive a database outage and still serve toggles in #550 - I guess there won't be many updates while the database is down, but being able to see what's configured still makes sense to me.

Reply:
Hmz, jeah we should still allow known addons to be alive even if DB goes down. The first addons will only be connected to actual changes being stored (webhooks etc), but I guess future addons might be independent of changes (e.g a GraphQL addon).